Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,383 @@
+    redeclare final package Medium=Medium,
+    final num=numChi,
+    final m_flow_nominal=mCW_flow_nominal,
+    final dp_nominal=dpCW_nominal,
+    final dpValve_nominal = dpCooTowVal_nominal,
+    final TAirInWB_nominal=TAirInWB_nominal,
+    final TWatIn_nominal=TCW_nominal,
+    final dT_nominal=dT_nominal,
+    final dTApp=dTApp,
+    final PFan_nominal=PFan_nominal,
+    final TMin=TMin)
+    "Cooling towers with bypass valve"
+    annotation (Placement(transformation(extent={{-40,80},{-20,100}})));
+  Buildings.Applications.DataCenters.ChillerCooled.Equipment.FlowMachine_y pumCHW(
+    redeclare final package Medium=Medium,
+    final per=fill(
+      perCHWPum,
+      numChi),
+    use_inputFilter=false,
+    yValve_start=fill(
+      1,
+      numChi),
+    final energyDynamics=energyDynamics,
+    final m_flow_nominal=mCHW_flow_nominal,
+    final dpValve_nominal=dpCHWPumVal_nominal,
+    final num=numChi)
+    "Chilled water pumps"
+    annotation (Placement(transformation(extent={{-50,-16},{-30,4}})));
+  Buildings.Applications.DataCenters.ChillerCooled.Equipment.FlowMachine_m pumCW(
+    redeclare final package Medium=Medium,
+    final per=fill(
+      perCWPum,
+      numChi),
+    final energyDynamics=energyDynamics,
+    final m_flow_nominal=mCW_flow_nominal,
+    final dpValve_nominal=dpCWPumVal_nominal,
+    final num=numChi)
+    "Condenser water pumps"
+    annotation (Placement(transformation(extent={{20,80},{40,100}})));
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage valByp(
+    redeclare final package Medium=Medium,
+    final allowFlowReversal=false,
+    final m_flow_nominal=mCHW_flow_nominal*0.05,
+    final dpValve_nominal=dpCHW_nominal)
+    "Chilled water bypass valve"
+    annotation (Placement(transformation(extent={{10,10},{-10,-10}},rotation=0,origin={-30,-90})));
+  Buildings.Fluid.Sensors.MassFlowRate senMasFloByp(
+    redeclare final package Medium=Medium)
+    "Chilled water bypass valve mass flow meter"
+    annotation (Placement(transformation(extent={{40,-80},{20,-100}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTCHWSup(
+    redeclare final package Medium=Medium,
+    final m_flow_nominal=mCHW_flow_nominal)
+    "Chilled water supply temperature"
+    annotation (Placement(transformation(extent={{140,-30},{160,-50}})));
+  Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChilledWaterPumpSpeed chiWatPumCon(
+    tWai=0,
+    final m_flow_nominal=mCHW_flow_nominal,
+    final dpSetPoi=dpSetPoi)
+    "Chilled water pump controller"
+    annotation (Placement(transformation(extent={{-140,-12},{-120,8}})));
+  Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChillerStage chiStaCon(
+    final tWai=tWai,
+    final QChi_nominal=QChi_nominal)
+    "Chiller staging controller"
+    annotation (Placement(transformation(extent={{-240,184},{-220,204}})));
+  Modelica.Blocks.Sources.RealExpression mPum_flow(
```

Comment:
Can you replace that with a `Buildings.Fluid.Sensors.MassFlowRate` that is inline with the pumps?

Reply:
Thank you for the suggestions, the MassFlowRate sensor has been added.